chain/events/filter/index_migrations.go

```diff
@@ -161,10 +153,13 @@ func migrationVersion3(ctx context.Context, tx *sql.Tx) error {
 // And then creating the following indices:
 //  1. an index on the event.tipset_key_cid column
 //  2. an index on the event.height column
-//  3. an index on the event.reverted column
-//  4. an index on the event_entry.indexed and event_entry.key columns
-//  5. an index on the event_entry.codec and event_entry.value columns
+//  3. an index on the event.reverted column (removed in version 7)
+//  4. an index on the event_entry.indexed and event_entry.key columns (removed in version 7)
```

Comment:
Better to document that we're removing composite indices here and below and not indices on the individual columns.